redme.txt

```diff
@@ -4,5 +4,4 @@ Idea : The game idea is very simple. It basically contains the famous instagram
 
 Features:1.There will be a login and signup page
 	 2.There will a leaderboard which will show the most cringiest reel.
-	 3. People can vote in reels to take it on leaderboard.
-	 4. User can write their username
\ No newline at end of file
```

Comment:
It's a good practice to end a file with a newline character. This is a POSIX standard and some tools might not work properly if this is not followed. [medium]